tests/test_vrt.py

```diff
@@ -40,7 +31,9 @@ def test_vrt_schema(self):
 
     def test_mapping_schemas(self):
         for mapping in self.mappings:
-            schema_file = os.path.join(utils.MAPPING_DIR, mapping['name'] + '.schema.json')
+            name_path = mapping['name'] + '/' + mapping['name'] if mapping['nested'] else mapping['name']
+
+            schema_file = os.path.join(utils.MAPPING_DIR, name_path + '.schema.json')
```

Comment:
An idea:

```python
schema_file = os.path.join(utils.MAPPING_dir, mapping['name'] if mapping['nested'] else '', mapping['name'] + '.schema.json)
```

That makes it more obvious how the path is constructed.

Reply:
I'm going to break this into two lines for readability